src/dex/spark/spark.ts

```diff
@@ -207,6 +207,10 @@ export class Spark
       destAmount,
       swapData,
       exchange,
+      undefined,
+      undefined,
+      undefined,
+      isSell && this.isDai(destToken),
```

Comment:
`skipApproval` is missing in getDexParam?

Reply:
fixed